external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -115,69 +116,72 @@ abstract class DockerJDBCIntegrationSuite extends SharedSparkSession with Eventu
   protected var jdbcUrl: String = _
 
   override def beforeAll(): Unit = {
```

Comment:
I would do:

```scala
override def beforeAll(): Unit = runIfTestsEnabled(s"Prepare for $this.getClass.getName") {
```

or

```scala
override def beforeAll(): Unit = runIfTestsEnabled(
    s"Prepare for $this.getClass.getName") {
```

to minimise the diff because of the indentation